roles/maas/tasks/initialize_region_rack.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Initialize MAAS Region + Rack Controller
+  when: inventory_hostname in groups['maas_region_rack_server'] and maas_install.failed == false and maas_install.changed == true
+  tags: install_maas
+  block:
+    - name: Disable timesyncd service
```

Comment:
I don't know what happened to my original comment, but as noted in slack, this is still not good enough.  Let's work on this together rather than going back and forth in review comments about it.